File: Java/Bryann Valderrama/Algoritmos de Ordenamiento/bubbleSort.java

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {

    static void bubbleSort(int arr[], int n) {
        if (n == 1) {
            return;
        }
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n - 1; j++) {
                if (arr[j] > arr[j + 1]) {
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String entrada[] = br.readLine().split(" ");
        int arr[] = new int[entrada.length];
        for (int i = 0; i < entrada.length; i++) {
            arr[i] = Integer.parseInt(entrada[i]);
        }
        bubbleSort(arr, arr.length);
        for (int i : arr) {
            System.out.print(i + " ");
        }
        System.out.println();
    }
}

File: PY/Bryann Valderrama/Algoritmos de Ordenamiento/bubbleSort.py
def bubbleSort(arr, n):
    if (n == 1):
        return
    for i in range(0, n):
        for idx in range(0, n-1):
            if (arr[idx] > arr[idx+1]):
                temp = arr[idx]
                arr[idx] = arr[idx+1]
                arr[idx+1] = temp


lista = [int(x) for x in input().split()]
bubbleSort(lista, len(lista))
print(*lista)
